elementary/messages/blocks.py

```diff
@@ -61,7 +61,22 @@ class MentionBlock(BaseInlineTextBlock):
     user: str
 
 
-InlineBlock = Union[TextBlock, LinkBlock, IconBlock, InlineCodeBlock, MentionBlock]
+class LineBlock(BaseBlock):
+    type: Literal["line"] = "line"
+    inlines: Sequence["InlineBlock"]
+    sep: str = " "
+
+
+InlineBlock = Union[
+    TextBlock,
+    LinkBlock,
+    IconBlock,
+    InlineCodeBlock,
+    MentionBlock,
+    "LineBlock",
+]
+
+LineBlock.update_forward_refs()
```

Comment:
needed to join a subsection of the line with a different sep (the owners with a ',')